pkg/dlx/handler.go

```diff
@@ -63,7 +67,14 @@ func (h *Handler) handleRequest(res http.ResponseWriter, req *http.Request) {
 			res.WriteHeader(http.StatusBadRequest)
 			return
 		}
-		targetURL, err = url.Parse(fmt.Sprintf("http://%s:%d/%s", resourceName, h.targetPort, path))
+		serviceName, err := h.resourceScaler.ResolveServiceName(scaler_types.Resource{Name: resourceName})
+		if err != nil {
+			h.logger.WarnWith("Failed resolving service name",
+				"err", errors.GetErrorStackString(err, 10))
```

Comment:
Can be one liner (fits imo)

Reply:
<img width="838" alt="Screen Shot 2020-04-17 at 3 28 34" src="https://user-images.githubusercontent.com/22135088/79519372-5931bd80-805c-11ea-9b89-594c84558e42.png">
Basically it fits one line, but with the IDE additions of the arg names it doesn't
so I think better leave it as is